frontend/src/components/custom-tools/combined-output/CombinedOutput.jsx

```diff
@@ -122,15 +143,41 @@ function CombinedOutput({ docId, setFilledFields }) {
       });
   };
 
+  const getAdapterInfo = () => {
+    axiosPrivate
+      .get(`/api/v1/unstract/${sessionDetails.orgId}/adapter/?adapter_type=LLM`)
+      .then((res) => {
+        const adapterList = res.data;
```

Comment:
@jagadeeswaran-zipstack Please use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here and wherever applicable to make sure we don't run into errors.